server/migrations/20210426072646-add-snapshot-to-chain.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('Chains', 'snapshot', { type: Sequelize.STRING });
```

Comment:
Do you have a plan on how to use this String to connect to snapshot? Is there a straight-forward way to connect to the appropriate snapshot space with the ENS domain?

Reply:
Yes, the snapshot string is just a URL string to call the snapshot apis.
For example, I added "ybaby.eth" as snapshot field for Yearn chain. So we can use snapshots apis for Yearn like "https://snapshot.org/#/ybaby.eth".
Not sure it would be a good idea later. But it makes sense for now.

Reply:
That makes sense to me as a `unique identifier`